CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+## Contributing
+
+[fork]: https://github.com/education/classroom/fork
+[pr]: https://github.com/education/classroom/compare
+[style]: https://github.com/bbatsov/ruby-style-guide
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+## Submitting a pull request
+
+0. [Fork][fork] and clone the repository
+0. Configure and install the dependencies: (See the [README](README.md) for more details)
```

Comment:
or `https://github.com/education/classroom/blob/readme/README.md#hacking-on-classroom-for-github`